products/products.css

```diff
@@ -0,0 +1,47 @@
+ul {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+li {
+    margin-top: 0;
+    background: #0c130f;
+    margin: 10px;
+    border: 4px solid #2c1e0a;
+    border-radius: 15px;
+    padding: 30px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    height: 300px;
+    width: 300px;;
+    box-shadow: 0 0 10px black;
+}
+
+img {
+    max-height: 200px;
+    height: auto;
+    width: 100%;
+    border-radius: 10px;
+}
+
+.price-and-add {
+    display: flex;
+    justify-content: space-around;
+    padding-top: 15px;
+}
+
+h3 {
+    margin-top: 0;
+    height: 10px;
+    border: 0;
+}
+
+hr {
+    height: 12px;
+    border: 0;
+    box-shadow: inset 0 12px 12px -12px rgba(170, 236, 203, 0.5);
+}
+
+/* li > */
```

Comment:
what's this one doin'?

Reply:
so I just figured out how to see comments in here :/ . This one brings a box-shadow inside of the element instead of outside and then the rgba gives an rgb color and the "a" sets the opacity :)